solr/core/src/test/org/apache/solr/api/ContainerPluginsRegistryTest.java

```diff
@@ -0,0 +1,105 @@
+package org.apache.solr.api;
+
+import java.nio.file.Path;
+import java.util.Map;
+import org.apache.solr.SolrTestCaseJ4;
+import org.apache.solr.cluster.placement.plugins.AffinityPlacementConfig;
+import org.apache.solr.core.CoreContainer;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/** Tests for {@link ContainerPluginsRegistry} */
+public class ContainerPluginsRegistryTest extends SolrTestCaseJ4 {
```

Comment:
There is an existing `TestContainerPlugin` class that tests the same area of code, but in a purely Solr Cloud context. Whether or not to merge both depends on whether or not we want to go with cluster or container plugins imho.

Reply:
Yes, I agree, that is why I did not reuse the existing test class, considering that the feature I was adding was not specific to Solr cloud. I would like as well to have another PR for the various renamings, if this one gets through.